### PR TITLE
Fixed #26458 -- Based Avg's default output_field resolution on its source field type.

### PR DESCRIPTION
**Pull request by charettes**

### CODE REVIEW

File: tests/aggregation/tests.py [hunk 1]
```diff
@@ -497,13 +497,13 @@ def test_aggregate_annotation(self):
 
     def test_avg_duration_field(self):
         self.assertEqual(
-            Publisher.objects.aggregate(Avg('duration', output_field=DurationField())),
+            Publisher.objects.aggregate(Avg('duration')),
             {'duration__avg': datetime.timedelta(days=1, hours=12)}
         )
```
Comment (charettes) [inline tests/aggregation/tests.py]: I could keep this explicit `output_field` version and add an explicit below if required.

**Comment by timgraham** [comment]
I'd like if you made the import change in a separate commit so those lines don't get blamed to an unrelated change.

**Comment by charettes** [comment]
> I'd like if you made the import change in a separate commit so those lines don't get blamed to an unrelated change.

Will do.

Any thoughts about the necessity to keep the explicit `output_field` version of the tests?

**Comment by timgraham** [comment]
Not sure

**Comment by charettes** [comment]
Buildbot, test on oracle.

**Comment by timgraham** [comment]
@jarshwah, any input?

**Comment by jarshwah** [comment]
LGTM. I think leaving a single test in that explicitly defines output_field is a decent idea, to protect from that kwarg being removed.

**Pull request merged**